components/02-content/accordion/accordion.js

```diff
@@ -19,11 +19,22 @@ module.exports = function () {
       const self = this;
       const header = options.hash.header || 'div';
 
+      var termsEnglish = ["success", "warning", "info", "danger"];
+      var termsDutch = ["succes", "waarschuwing", "info", "gevaar"];
+      // var stateTerm = termsEnglish.indexOf(self.state);
+      // var dutchTerm = termsDutch.slice(stateTerm, stateTerm + 1);
+      var dutchTerm = termsDutch[termsEnglish.indexOf(self.state)]
+
       let html = (
         `<div class="dso-accordion-section${(self.state ? ' dso-' + self.state : '') + (self.open ? ' dso-open' : '') + (self.sections ? ' dso-nested-accordion' : '') }">
           <${header} class="dso-section-handle">
-            <a href="#">${ title }</a>
-          </${header}>`);
+            <a href="#">`);
+      if (dutchTerm) {
+        html += (`<span class="sr-only">${dutchTerm}: </span>`);
+      }
+      html += (`
+        ${ title }</a>
```

Comment:
* Whitespaces in accolades weg
* Whitespaces voor de `$` weg, zodat het weer op 1 regel komt
* Lege regel na eind-accolade